### ISSUE TITLE
Connection failed: Column metric must be of type char,varchar or text, got: *uint64 0xc42081f2b8

### ISSUE THREAD

**Issue by vukomir** [alexanderzobnin/grafana-zabbix#569]
### I'm submitting a ...
- [X ] Bug report

_If you don't send bug report, remove sections below_
### For bug report please include this information:

**Software Versions**

| Grafana | Zabbix | Grafana-Zabbix Plugin |
| ------- | ------ | --------------------- |
| 5.1.1   | 3.4.7  | 3.9.1                 |

#### What did you do?
Updated grfana to last version

#### What was the expected result?
Normal operation

#### What happened instead?
after update zabbix data stoped apearing in Grafana, when i tested the connection(Data Sources) i got following error.
Connection failed: Column metric must be of type char,varchar or text, got: *uint64 0xc42081f2b8
this only hapens when Direct DB Connection is enable in Zabbix Data Sources

**Comment by jtl999** [comment]
I can confirm the same issue

**Comment by plrunner** [comment]
+1

zabbix-server-mysql: 3.4.9
grafana: 5.1.3
grafana-zabbix: 3.9.1

**Comment by plrunner** [comment]
To work around the problem I just downgraded Grafana to 5.1.0 and got it working again.

**Comment by alexanderzobnin** [comment]
This should be fixed, see #564. Could you show network request when you saving dashboard?

**Comment by alexanderzobnin** [comment]
@noonedeadpunk you can use browser dev tools to trace network queries. Please, post SQL query invoked by grafana.

**Comment by noonedeadpunk** [comment]
It's pretty strange, but today the problem has resolved itself... Probably, there had been some browser cache... But before posting reply in the thread I've restarted grafana server several times and reloaded page.

So it seems, that problem has been fixed in 3.9.1.
The SQL request, that I've found today was the following:
`{"queries":[{"refId":"A","format":"time_series","datasourceId":3,"rawSql":"SELECT CAST(itemid AS CHAR) AS metric, clock AS time_sec, value_avg AS value FROM trends_uint LIMIT 1","maxDataPoints":10000}]}`
But it seems, that it doesn't matter anymore.
